ddprof-exporter/Cargo.toml

```diff
@@ -3,7 +3,7 @@
 
 [package]
 name = "ddprof-exporter"
-version = "0.3.0"
+version = "0.4.0-rc.1"
```

Comment:
Complete nit, but is tokenizing release candidates by `.` a convention?  If not, could I propose using a different delimiter than what we use for `major.minor.path` version numbers?  The reason is, it's nice for folks validating a version (potentially in various languages) when a delimiter separates tokens of uniform syntactical meaning.

Again, total nit.

Reply:
It's part of [semver](https://semver.org/), which is why I chose that originally. I imagine Erwan is just following what I did.

Reply:
Yes. I will discuss releasing process at next status. If you have things in mind we would like to improve, feel free to mention these.